src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -97,10 +97,17 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
                 if (searchElement.CreateAndConnectCommand.CanExecute(port.PortModel))
                 {
                     searchElement.CreateAndConnectCommand.Execute(port.PortModel);
+                    var nodeName = (searchElement.Model is Search.SearchElements.NodeModelSearchElement) ? searchElement.FullName : searchElement.Model.CreationName;
+                    var searchElementInfo = ViewModel.IsDisplayingMLRecommendation ?
+                        nodeName + " " + port.PortModel.Index.ToString() + " " + port.PortName + " " + port.NodeViewModel.OriginalName + " " +
```

Comment:
Same 2 cases for the target node where the autocomplete is triggered. Use port.NodeViewModel.NodeModel.CreationName for zerotouch nodes.